README.md

```diff
@@ -0,0 +1,32 @@
+# Flights Application
+
+Software application that is able to request all flight data from two APIs (<https://discovery-stub.comtravo.com/source1>) and (<https://discovery-stub.comtravo.com/source2>) and then it will send back to customer a response in JSON format of all flights but deleting duplicates in the process.
+
+### Setup 
+* Open a terminal in the root folder of this project and run `npm install` in order to install all dependencies of the package.json.
+* Save  the following environment variables in a `config.env` file:
+
+```shell
+LINK_ONE=https://discovery-stub.comtravo.com/source1
+LINK_TWO=https://discovery-stub.comtravo.com/source2
+USERNAME_API=ct_interviewee
+PASSWORD_API=supersecret
+```
+
+* In the same terminal run `npm run start`.
+* Make a GET request in the only endpoint of the project (<http://localhost:3000/flights>) to request all data from both APIs, it will sent you a response of all flights sorted without any duplicates.
+
+## Testing
+
+* Run `npm run test` and it will run all test that are storaged on `/test` folder
+
+## Libraries / Dependencies
+
+- [Axios](https://github.com/axios/axios) - Used to fetch flights from APIs
+- [Dotenv](https://www.npmjs.com/package/dotenv) - Used to storage environment variables in a .env file 
+- [Express](https://expressjs.com/) - Used for structure of project
```

Comment:
I think Express is doing a little more than only giving structure to the project 😅

Reply:
hahahaha what do you think I should write there?

Reply:
I would say you're using it as web framework to serve the flights endpoint.